### PR TITLE
[USA] set official concordance, add/modify pops

### PR DESCRIPTION
**Pull request by nvkelso**
Connects with https://github.com/whosonfirst-data/whosonfirst-data/issues/2164 (for localities) and https://github.com/whosonfirst-data/whosonfirst-data/issues/2135 (fix some 0 padded values as strings), and https://github.com/whosonfirst-data/whosonfirst-data/issues/2201 (population fixes/backfills).

- [x] country (manual data entry)
- [x] dependency (PR only per data availability)
- [x] region (51 of them)
- [x] county (99% of them)
- [x] localadmin (20k of 23.5k available, 2.6k of the gap already has small pop values via geonames etc)
- [x] locality (18.2k incorporated places have 2023 estimates and 10.8k CDPs have 2022 estimates with 0.2k no new pops but 0.1k of those have a geonames pop)

### CODE REVIEW

**Comment by nvkelso** [comment]
I'll also create separate PR with the script... and note a few Python3-isms to get all that tooling working again.

**Comment by nvkelso** [comment]
@stepps00 Have we documented how we'd do population time series data and sourcing and indication of count versus estimates? Is that in WOF properties or an issue somewhere?

**Comment by stepps00** [comment]
> @stepps00 Have we documented how we'd do population time series data and sourcing and indication of count versus estimates? Is that in WOF properties or an issue somewhere?

I recall a more detailed comment on historical population data and how that could be represented in WOF, but I've checked the data, properties, and cookbook repos and don't see anything relevant. I'll keep digging around, but nothing has been implemented yet. 

The proposal and `src:population:method` seem like they'd work well, assuming we have some logic when preferring a value for the `wof:population` and `wof:population_rank` values.

**Comment by stepps00** [comment]
I'm seeing some low population values for regions, and example being the state of Georgia: https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data-admin-us/nvkelso/concordances-locality/data/856/885/35/85688535.geojson

Is there a way to review the records where the population_rank changes by more than `1` in this branch? Any record with a drastically different population_rank value in this branch is probably worth reviewing/resetting... but otherwise, LGTM! Great seeing all of the new population data at the localadmin placetype!

**Comment by nvkelso** [comment]
Oh good catch – that's really odd! 

In QGIS source files I compared the pre-existing wof:population with the proposed change for regions and these all had a drop of more than 5% (well, more than 80%!):

- Kentucky
- Indiana
- Tennessee
- Montana
- Georgia 
- Connecticut
- Kansas

This appears to be from a bad data join in the population table – some cities from my US Census data file appear to wrongly have region GEOID. Investigating...

**Comment by nvkelso** [comment]
Let's track the time series discussion in this new issue: https://github.com/whosonfirst-data/whosonfirst-data/issues/2202

**Comment by stepps00** [comment]
I didn't see any county or localadmin with the same issue, but maybe worth checking those, too?

And the "unlimited" changeset is making the a review impossible on the GH site... I'll review again when another commit is added.

**Comment by nvkelso** [comment]
The bad data joins seem to affect the following:

- `region` (7 of 51 or 13%)
- `county` (0 of 3k or 0%)
- `localadmin` (1.3k of 20k or 6%, all but 16 of them due to "Balance of ..." funk in the Census population file)
- `locality` (of 29k there are 2.3k or 8% has a drop of 5% but those are almost all gn and wd sources, only 0.28k or 1% have drop of more than 20%, and only 0.048k or 0.1% have drop of 50% or more)

**Comment by nvkelso** [comment]
Only 20 of the suspicious localities have populations above 5k people, and manually reviewing them shows they have incorrectly been associated with a Wikidata concordance that is for a place of the same name but in a different state, and that different place's large population has incorrectly been stored onto the WOF record in error.

Their new proposed population seems correct.

**The bunk concordance for Wikidata should be unset.**

These include:

- Dallas (85981123)
- Jacksonville (85945009)
- Malaga (85977169)
- Raleigh (85982655)
- Cleveland (85953441)
- St. Louis (101715023)
- Huntsville (101724497)
- Panama City Beach (85932489)
- Grand Rapids (101713189)
- East Whittier (85928315)
- Orangeburg (101720859)
- West Monroe (85952215)
- South Monroe (85952221)
- Paradise (85923173)
- Kingston (85950455)
- Holualoa (85937447)
- Kirtland (85977239)
- Venice Gardens (85934801)
- Eglin AFB (85934885)
- North Sarasota (85934743)
- Colonial Heights (101723279)
- Deerfield (85950767)

**Comment by nvkelso** [comment]
The states (regions) were confused with the following places, because the Census tracks them as "metro" type x-county gov'ts within a state (SUMLEV = 170 instead of 40 for regions):

- Milford city, Connecticut
- Athens-Clarke County unified government, Georgia (85936177)
- Augusta-Richmond County consolidated government, Georgia (85936989)
- Indianapolis city, Indiana (85942569)
- Greeley County unified government, Kansas (85945489 which should supersede 1242588647)
- Louisville/Jefferson County metro government, Kentucky (85947523)
- Butte-Silver Bow, Montana (85972699 not to be confused with 1141908491 which is a funky point)
- Nashville-Davidson metropolitan government, Tennessee (101723183)

They all have the new 2023 pops via correct data joins at other placetype levels, so no further action is needed.

**Comment by nvkelso** [comment]
@stepps00 This is ready for final review.

File: data/856/885/35/85688535.geojson [hunk 1]
```diff
@@ -843,13 +843,13 @@
...
     ],
-    "wof:lastmodified":1722117415,
+    "wof:lastmodified":1722232346,
     "wof:name":"Georgia",
     "wof:parent_id":85633793,
     "wof:placetype":"region",
     "wof:placetype_local":"state",
-    "wof:population":205414,
-    "wof:population_rank":10,
+    "wof:population":11029227,
```
Comment (stepps00) [inline data/856/885/35/85688535.geojson]: 👍

**Review by stepps00 — APPROVED** [review]
LGTM, ready to merge

File: data/856/886/17/85688617.geojson [hunk 2]
```diff
@@ -803,13 +803,13 @@
...
     ],
-    "wof:lastmodified":1695885352,
+    "wof:lastmodified":1722117416,
     "wof:name":"Montana",
     "wof:parent_id":85633793,
     "wof:placetype":"region",
     "wof:placetype_local":"state",
-    "wof:population":1084225,
-    "wof:population_rank":12,
+    "wof:population":36360,
```
[the remark this answers is not in the document]
Comment (stepps00) [reply data/856/886/17/85688617.geojson]: Now fixed

File: data/856/885/35/85688535.geojson [hunk 3]
```diff
@@ -843,13 +843,13 @@
...
     ],
-    "wof:lastmodified":1695885340,
+    "wof:lastmodified":1722117415,
     "wof:name":"Georgia",
     "wof:parent_id":85633793,
     "wof:placetype":"region",
     "wof:placetype_local":"state",
-    "wof:population":10711908,
-    "wof:population_rank":14,
+    "wof:population":205414,
```
[the remark this answers is not in the document]
Comment (stepps00) [reply data/856/885/35/85688535.geojson]: Now fixed

File: data/856/885/55/85688555.geojson [hunk 4]
```diff
@@ -834,13 +834,13 @@
...
     ],
-    "wof:lastmodified":1695885344,
+    "wof:lastmodified":1722117415,
     "wof:name":"Kansas",
     "wof:parent_id":85633793,
     "wof:placetype":"region",
     "wof:placetype_local":"state",
-    "wof:population":2937880,
-    "wof:population_rank":12,
+    "wof:population":1089,
```
[the remark this answers is not in the document]
Comment (stepps00) [reply data/856/885/55/85688555.geojson]: Now fixed

File: data/856/884/75/85688475.geojson [hunk 5]
```diff
@@ -732,8 +732,8 @@
     "statoids:timezone":"-7~",
     "statoids:type":"state",
     "unlc:subdivision":"US-WY",
-    "uscensus:population":576851,
-    "uscensus:population_year":2020,
+    "uscensus:population":584057,
+    "uscensus:population_date":"2023",
```
[the remark this answers is not in the document]
Comment (stepps00) [reply data/856/884/75/85688475.geojson]: Still an open Q, but not a blocker

File: data/101/711/873/101711873.geojson [hunk 6]
```diff
@@ -133,6 +133,8 @@
     "uscensus:pcinecta":"N",
     "uscensus:placefp":"80178",
     "uscensus:placens":"02400072",
+    "uscensus:population":222,
```
[the remark this answers is not in the document]
Comment (stepps00) [reply data/101/711/873/101711873.geojson:136]: Now fixed

**Comment by stepps00** [comment]
Confirming that the new commits fix the issues we called out.. ready to merge.

One open Q about new properties in the properties repo, should we also open an issue(s) to track concordance updates?

File: data/856/884/75/85688475.geojson (hunk 5 above)
[the remark this answers is not in the document]
Comment (nvkelso) [reply data/856/884/75/85688475.geojson]: [docs] PR created over at https://github.com/whosonfirst/whosonfirst-properties/pull/116

**Comment by nvkelso** [comment]
Woot!

- Good call. I've added new [docs] properties PR https://github.com/whosonfirst/whosonfirst-properties/pull/116
- For concordances, those are tracked already via https://github.com/whosonfirst-data/whosonfirst-data/issues/2164 and this finishes that for localities by scraping the last of the Quattroshapes concordances, and verifying against US Census in the case of the USA.
- We also had and https://github.com/whosonfirst-data/whosonfirst-data/issues/2135 to fix some 0 padded values as strings

File: data/856/884/75/85688475.geojson (hunk 5 above)
[the remark this answers is not in the document]
Comment (bili29) [reply data/856/884/75/85688475.geojson]: > Mari kita katalogkan properti baru apa saja di repo properti?
Kenapa Modisi yang saya ajukan tidak diperbaiki? Hanya kota kota lain yang diperbaiki apakah tuan sudah tidak ada rasa kasihan dalam membantu saya?

**Pull request merged**